### ISSUE TITLE
nuxt 3 support?

### ISSUE THREAD

**Issue by GamerGirlandCo** [nickfrosty/nuxt-seo#14]

**Comment by nickfrosty** [comment]
@GamerGirlandCo At this time, I do not plan on making major updates. But PRs are welcome :)

**Comment by GamerGirlandCo** [comment]
> @GamerGirlandCo At this time, I do not plan on making major updates. But PRs are welcome :)

fair enough. though i did figure a solution out by myself, i'll try to open a PR when i can

**Comment by aayushali** [comment]
@GamerGirlandCo Can you please share a code snippet on how you achieved Dynamic SEO in Nuxt 3?
I am trying to get dyanamic value for the Meta but could not achieve it.
